tests/python/frontend/pytorch/test_forward.py

```diff
@@ -363,9 +363,35 @@ class MaxPool2D2(Module):
         def forward(self, *args):
             return torch.nn.MaxPool2d(kernel_size=[10, 10])(args[0])
 
+    class MaxPool2D3(Module):
```

Comment:
Remove this wrapper

Reply:
Don't we need a test for padding and stride in Maxpool?

Reply:
Don't we need a test for padding and stride in Maxpool?

Reply:
I am talking about the same point as https://github.com/apache/incubator-tvm/pull/5142/files#r397046667

Yes, you should have a test, but no need to write a wrapper class

Reply:
> I am talking about the same point as https://github.com/apache/incubator-tvm/pull/5142/files#r397046667
> 
> Yes, you should have a test, but no need to write a wrapper class

Thanks. I misunderstood your review suggestion :-)

Reply:
done.